### DISCUSSION TITLE
Jekyll pages/al-folio theme

### DISCUSSION THREAD

**Discussion by alexislozano09** [alshedivat/al-folio#1327] [Q&A] [answered]
Hey guys, I have managed to remove all posts on the blog section apart from "Displaying External Posts on Your al-folio Blog" & "A post with code" I cannot find where the files or values are for these two. Any suggestions?

**Comment by maxxxzdn** [comment, marked as answer]
you have to delete the link to medium here:

https://github.com/alshedivat/al-folio/blob/ee3f2b9dabcc7da79229565ef71f7532a3af88b9/_config.yml#L159

**Comment by muhammadfikri13** [comment]
is this still works?

**Reply by george-gca** [reply]
Have you tried it?

**Comment by alexislozano09** [comment]
Yes, set up on Linux.

**Reply by george-gca** [reply]
Did it not work?
